gitium/functions.php

```diff
@@ -206,10 +206,50 @@ function gitium_commit_and_push_gitignore_file( $path = '' ) {
 	gitium_merge_and_push( $commit );
 }
 
+if ( ! function_exists( 'gitium_before_merge_and_push' ) ) :
```

Comment:
We should rename this function to something more appropriate, like `gitium_acquire_merge_lock` and it should return the lock handle. So the code in `gitium_merge_and_push` whould look something like:
```php
$lock = gitium_acquire_merge_lock();
...
gitium_release_merge_lock($lock);
```